src/dmd/lexer.d

```diff
@@ -201,8 +202,22 @@ class Lexer
         return peek(t).value;
     }
 
+    /****************************
+     * Turn next token in buffer into a token, including newlines.
+     * Params:
+     *  t = the token to set the resulting Token to
+     */
+    final void scanUntilNewline(Token* t)
+    {
+        this.tokenizeNewlines++;
+        scan(t);
+        this.tokenizeNewlines--;
```

Comment:
Bikeshed topic, whether to just make this field a `bool` parameter of scan() or not. I'm on the fence either way.